lib/ansible/modules/find.py

```diff
@@ -154,6 +154,14 @@
             - When doing a C(contains) search, determine the encoding of the files to be searched.
         type: str
         version_added: "2.17"
+    limit:
+        description:
+            - Limit the maximum number of matching paths returned. After finding this many, the find action will stop looking.
+            - Matches are made from the top, down (i.e. shallowest directory first).
+            - Set to V(None) for unlimited matches.
```

Comment:
`If not set, or set to v(null), it will do unlimited matches`

Reply:
```suggestion
            - If not set, or set to v(null), it will do unlimited matches
```

Reply:
Changed